docs/gettingstarted.md

```diff
@@ -128,19 +133,24 @@ Process Identification
 ----------------------
 
 At the crux of AquaQ TorQ is how processes identify themselves. This is
-defined by two variables - .proc.proctype and .proc.procname which are
-the type and name of the process respectively. These two values
-determine the code base and configuration loaded, and how they are
-connected to by other processes. If both of these are not defined, the
-TorQ will attempt to use the port number a process was started on to
-determine the code base and configuration loaded.
+defined by two required variables - .proc.proctype and .proc.procname
+which are the type and name of the process respectively and an optional

```

Comment:
Can you make this a new sentence (after respectively). Something like “An optional variable parentproctype allows an initial codebase and configuration to be loaded”